content/v2.0/security/enable-https.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Enable HTTPS with InfluxDB
```

Comment:
Sorry to keep coming back to this. I've looked at a few related products and they're documentation around enabling TLS/HTTPS. Let's do the following:

- Update the `title` to "Enable TLS encryption"
- Update the file name to `enable-tls.md`
- In the frontmatter, just under `title`, add `seotitle: Enable TLS/SSL encryption`